src/esp/assets/ResourceManager.cpp

```diff
@@ -101,6 +112,32 @@ bool ResourceManager::loadScene(const AssetInfo& info,
   Magnum::Trade::MeshData3D cube = Magnum::Primitives::cubeWireframe();
   primitive_meshes_.push_back(Magnum::MeshTools::compile(cube));
 
+  // compute the absolute transformation for each static drawables
+  if (meshSuccess && parent && computeAbsoluteAABBs_) {
+    if (info.type == AssetType::FRL_PTEX_MESH) {
+      // retrieve the ptex mesh data
+      const std::string& filename = info.filepath;
+      CORRADE_ASSERT(resourceDict_.count(filename) != 0,
+                     "ResourceManager::loadScene: ptex mesh is not loaded.",
+                     false);
+      MeshMetaData& metaData = resourceDict_.at(filename);
+      CORRADE_ASSERT(
+          metaData.meshIndex.first == metaData.meshIndex.second,
+          "ResourceManager::loadScene: ptex mesh is not loaded correctly.",
+          false);
+
+      computePTexMeshAbsoluteAABBs(*(meshes_[metaData.meshIndex.first].get()));
+    } else if (info.type == AssetType::MP3D_MESH) {
+      computeGeneralMeshAbsoluteAABBs();
+    }
+  }
+
+  if (computeAbsoluteAABBs_) {
+    computeAbsoluteAABBs_ = false;
+    // this is to prevent it from being misused in the future
+    staticDrawableInfo_.clear();
```

Comment:
@erikwijmans : I addressed your concern here, not at the beginning of this function.
Every time, after the job is done, it will be cleared.

Reply:
Yeah, I saw this.  I was thinking a clear at the top may also be useful for readability.  That class member is really just a local variable to this function.

Reply:
> That class member is really just a local variable

Yes, correct. I made it a member variable to avoid long and tedious argument list for a couple of member functions.
But will it make you more comfortable (putting it in the function arguments)?

Reply:
@erikwijmans : I tried to put this "local" variable into the argument list for a couple of related functions. It looked very ugly, and thus significantly decreased the code readability. I will have to give it up.

I will put "a clear at the top" to address your concern.